src/main/java/core/basesyntax/service/impl/ReportGeneratorServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportGeneratorService;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ReportGeneratorServiceImpl implements ReportGeneratorService {
+    private StorageDao storageDao = new StorageDaoImpl();
+
+    @Override
+    public String report() {
+        List<String> report = new ArrayList<>();
+        report.add("fruit,quantity");
+        for (Map.Entry<Fruit, Integer> entry : storageDao.getStorage().entrySet()) {
```

Comment:
```suggestion
        for (Map.Entry<Fruit, Integer> entry : storageDao.getAll()) {
```
let's make some getAll method which will return entry set from storage